Listener/LicenseListener.php

```diff
@@ -57,16 +58,19 @@ public function onKernelRequest(GetResponseEvent $event)
             // Checking for whitelisted users
             try {
                 $user = $this->tokenStorage->getToken()->getUser();
-                $today = date('Y-m-d');
-                if ($user instanceof UserInterface) {
+                if ($user instanceof Tenant) {
+                    if($user->isWhiteListed()){
+                        return;
+                    }
+
+                    $today = date('Y-m-d');
                     $whitelist = $this->kernel->getContainer()->getParameter('license_whitelist');
                     foreach ($whitelist as $allowed) {
```

Comment:
Thanks, I like the idea of being able to manipulate this from the DB!

Could you please cleanup the old logic of checking the params, as well as remove the relevant entries from `parameters.yml.dist`?